### PR TITLE
Fixes for windows replay

### PR DESCRIPTION
**Pull request by AWoloszyn**
Based on #136.

### CODE REVIEW

**Comment by Qining** [comment]
lgtm

File: gapii/vulkan/vk_graphics_spy/cc/CMakeBuild.cmake [hunk 1]
```diff
@@ -31,7 +31,7 @@ foreach(abi ${ANDROID_ACTIVE_ABI_LIST})
     )
 endforeach()
 
-if(NOT DISABLED_CXX AND NOT WIN32) # TODO: Windows build not currently supported
+if(NOT DISABLED_CXX) # TODO: Windows build not currently supported
```
Comment (ben-clayton) [inline gapii/vulkan/vk_graphics_spy/cc/CMakeBuild.cmake]: Remove TODO?

File: gapii/vulkan/vk_graphics_spy/cc/layer.cpp [hunk 2]
```diff
@@ -14,30 +14,84 @@
...
   if (fn != nullptr) \
     return fn
 
-extern "C" {
+HMODULE LoadGAPIIDLL() {
+  char path[MAX_PATH] = { '\0' };
+  HMODULE this_module = GetModuleHandle("libVkLayer_GraphicsSpy.dll");
+  const char libgapii[] = "libgapii.dll";
+  if(this_module == NULL) {
+    return 0;
```
Comment (ben-clayton) [inline gapii/vulkan/vk_graphics_spy/cc/layer.cpp]: Probably worth logging an error for each of the failure cases.

File: gapii/vulkan/vk_graphics_spy/cc/CMakeBuild.cmake (hunk 1 above)
Comment (AWoloszyn) [inline gapii/vulkan/vk_graphics_spy/cc/CMakeBuild.cmake]: Done

File: gapii/vulkan/vk_graphics_spy/cc/layer.cpp (hunk 2 above)
Comment (AWoloszyn) [inline gapii/vulkan/vk_graphics_spy/cc/layer.cpp]: Done

**Pull request merged**